src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -692,56 +705,75 @@
           "type": "string"
         },
 
-        "fullName": {
-          "description": "The name of the tool along with its version and any other useful identifying information, such as its locale.",
+        "startTime": {
+          "description": "The date and time at which the run started. See \"Date/time properties\" in the SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "endTime": {
+          "description": "The date and time at which the run ended. See \"Date/time properties\" in the  SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "correlationId": {
+          "description": "An identifier that allows the run to be correlated with other artifacts produced by a larger automation process.",
           "type": "string"
         },
 
-        "version": {
-          "description": "The tool version.",
+        "architecture": {
+          "description": "The hardware architecture for which the run was targeted.",
           "type": "string"
         },
 
-        "semanticVersion": {
-          "description": "The tool version rendered as Semantic Versioning 2.0.",
+        "processId": {
+          "description": "The process identifier for the analysis tool run.",
           "type": "string"
         },
 
-        "fileVersion": {
-          "description": "The binary version of the tool's primary executable file (for operating systems such as Windows that provide that information).",
+        "workingDirectory": {
+          "description": "The working directory for the analysis rool run.",
           "type": "string",
-          "pattern": "[0-9]+(\\.[0-9]+){3}"
+          "default": { }
         },
 
-        "properties": {
-          "description": "Key/value pairs that provide additional information about the tool.",
+        "environmentVariables": {
+          "description": "The environment variables associated with the analysis tool process, expressed as key/value pairs.",
           "type": "object",
           "default": { }
         },
 
+        "properties": {
+          "description": "Key/value pairs that provide additional information about the run.",
+          "type": "object",
+          "default": { }
+        },       
+
         "tags": {
-          "description": "A set of distinct strings that provide additional information about the tool.",
+          "description": "A set of distinct strings that provide additional information about the run.",
           "type": "array",
           "uniqueItems": true,
           "items": {
             "type": "string",
             "default": [ ]
           }
-        }
-      },
-      "required": [ "name" ]
-    }
+        },
+        "required": { [ "name" ] }
```

Comment:
This change is wrong... not sure how it happened (since it doesn't look like something you'd've done by hand). The value of `required` must be an array of string: `"required": [ "name" ]`.

But... what is this doing here at all? `invocation` doesn't have a property called "`name`".